playbooks/roles/common/vars/main.yml

```diff
@@ -1,5 +1,8 @@
 ---
 streisand_common_packages:
+  # Required for generating passwords (not pre-installed on Ubuntu minimal)
+  - dictionaries-common
+  - wamerican-huge
```

Comment:
Interesting. When I was running streisand on an ubuntu-minimal install, apt was bailing on this. I had to uninstall the dictionaries-common package then reinstall it, after which it all worked. It could be that user input was required. I'll try it again and post more info